spec/factories/hardware.rb

```diff
@@ -12,6 +12,12 @@
       cpu_total_cores      4
     end
 
+    trait(:cpu2x1) do
```

Comment:
I'm completely in favor of using traits to set these values.  I would usually think Sockets x Cores, but maybe that's just me.  Anyone else have an opinion?